dd-java-agent/instrumentation/play-2.3/src/test/scala/server/SyncServer.scala

```diff
@@ -0,0 +1,32 @@
+package server
+
+import datadog.trace.agent.test.base.HttpServerTest
+import datadog.trace.agent.test.base.HttpServerTest.ServerEndpoint._
+import play.api.mvc.{Action, Handler, Results}
+import play.api.test.{FakeApplication, TestServer}
+
+object SyncServer {
+  val routes: PartialFunction[(String, String), Handler] = {
+    case ("GET", "/success") => Action { request =>
+      HttpServerTest.controller(SUCCESS, new ControllerClosureAdapter(Results.Status(SUCCESS.getStatus).apply(SUCCESS.getBody)))
+    }
+    case ("GET", "/redirect") => Action { request =>
+      HttpServerTest.controller(REDIRECT, new ControllerClosureAdapter(Results.Redirect(REDIRECT.getBody, REDIRECT.getStatus)))
+    }
+    case ("GET", "/query") => Action { request =>
+      HttpServerTest.controller(QUERY_PARAM, new ControllerClosureAdapter(Results.Status(QUERY_PARAM.getStatus).apply(QUERY_PARAM.getBody)))
+    }
+    case ("GET", "/error-status") => Action { request =>
+      HttpServerTest.controller(QUERY_PARAM, new ControllerClosureAdapter(Results.Status(ERROR.getStatus).apply(ERROR.getBody)))
```

Comment:
Hmmm. This test shouldn’t be passing 

```suggestion
      HttpServerTest.controller(ERROR, new ControllerClosureAdapter(Results.Status(ERROR.getStatus).apply(ERROR.getBody)))
```

Reply:
eh, it only matters for `NOT_FOUND`